docker/archlinux/base/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM archlinux:base
+
+# lkp install generates hosts/$HOSTNAME during create_host_config. Here it requires
+# user to pass hostname arg to specify a consistent name instead of the container
+# hostname when building the image.
+ARG hostname=lkp-docker
+ENV HOSTNAME=${hostname}
+
+ARG user=arch
```

Comment:
thanks, i found some existing code like
```
# yay not support root to build package
# Use regular user to do this
regular_user=$(grep 1000 /etc/passwd | awk -F':' '{ print $1 }')

sudo -u $regular_user yay -S --noconfirm --needed $*
```
Curious whether such code can be used to find a regular user instead of creating arch user.

Reply:
Does lkp user need to have uid/gid 1000? Here's the results of /etc/passwd:
![image](https://user-images.githubusercontent.com/31011189/235284892-66bb657e-1e3f-4bcd-b445-c2339358e495.png)

Initially when building the Dockerfile there's no user with uid 1000, so useradd arch will create user arch with uid 1000.

Should we get lkp user to be the one running the yay command?

Also, could you look into my previous comment as well? Much appreciated. Thank you!

Reply:
> Should lkp user have uid/gid 1000? Or should we get lkp user to be the one running the yay command?

Yes, to have lkp user as 1000 is preferred, and to use lkp user for later installation if feasible.

Reply:
Fixed. lkp user now at uid 1000, and used for later installation as well.
![image](https://user-images.githubusercontent.com/31011189/235346610-cf79c282-211e-4e20-8bd8-b3044e8b98c1.png)

Solution is creating lkp user instead of arch user for the yay installation, then subsequent `lkp install` will pick it up.